.github/workflows/ci.yml

```diff
@@ -0,0 +1,127 @@
+name: Continuous Integration
+
+on:
+  push:
+    branches:
+    - master
+
+  pull_request:
+    branches:
+    - master
+
+jobs:
+  coverage:
+    runs-on: ubuntu-latest
```

Comment:
Maybe set this to a fixed version and bump up manually to avoid spurious CI failures?

Reply:
The version doesn't change often and even when it does change you are warned months before GitHub changes what the `latest` one points to. You can find the images and related news [here](https://github.com/actions/virtual-environments).

Reply:
Alright, I guess it is not too much to ask of a maintainer to keep an eye open for this every second year ;)